.github/workflows/deploy.yml

```diff
@@ -0,0 +1,27 @@
+name: "Deploy to PyPI"
+on:
+  workflow_dispatch:
+    inputs: 
+      tag:
+        description: "Git tag to deploy to PyPI"
+        required: true
+
+jobs:
+  deploy:
+    name: "Deploy to PyPI"
+    runs-on: ubuntu-latest
+    environment: Deploy
+    steps:
+      - uses: actions/checkout@v1
+        with:
+          ref: ${{ github.event.inputs.tag }}
+      - uses: actions/setup-python@v2
+        with:
+          python-version: 3.9
```

Comment:
Picky hat on : I'd rather not fix Python version here. Always relying on stable should be enough for this need.

Reply:
TIL `3.x` is supported, I've got a couple places I can update now.